### DISCUSSION TITLE
[@types/chrome] ChromeSetting get/set/clear should support Promises.

### DISCUSSION THREAD

**Discussion by Jimmy-Z** [DefinitelyTyped/DefinitelyTyped#71075] [Issues with a @types package]
According to this, Promises are supported since Chrome 96 with MV3
https://developer.chrome.com/docs/extensions/reference/api/types#type-ChromeSetting

However
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/05c66f3f7cbb26073ff4f43d3af41708cfa64fa6/types/chrome/index.d.ts#L11621-L11641

Sorry my TS knowledge is not enough to fix this.

[a comment by an automated account is not shown]

**Comment by erwanjugand** [comment]
Relates to https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71269
